src/tv2-common/hotkeys/local.ts

```diff
@@ -11,15 +11,29 @@ function localSourceFullAudioHotkeyId(showStyleId: string, sourceLayer: string,
 }
 
 function localSourceFullAudioHotkeyName(source: string) {
-	return `EVS ${source}`
+	return `EVS ${source} 100%`
+}
+
+function localSourceFullAudioPick(sourceIndex: number, hotkeyType: string): number {
+	if (hotkeyType === 'queue' || /^cut_to_box_\d+$/.test(hotkeyType)) {
+		return 2 * sourceIndex
+	}
+	return sourceIndex
 }
 
 function localSourceVoAudioHotkeyId(showStyleId: string, sourceLayer: string, hotkeyType: string, index: number) {
 	return `${showStyleId}_${sourceLayer}_local_vo_audio_${hotkeyType}_${index}`
 }
 
 function localSourceVoAudioHotkeyName(source: string) {
-	return `EVS ${source}`
+	return `EVS ${source} VO`
+}
+
+function localSourceVoAudioPick(sourceIndex: number, hotkeyType: string): number {
```

Comment:
I don't really understand what this function does. What is the condition and where does the magic numbers come from and why!?

Reply:
I assume that you are referring to `sourceIndex` in `localSourceVoAudioPick`.
If we use the following snippet as our example, then the index of the shortcut is used as the sourceIndex to
https://github.com/tv2/sofie-blueprints-inews/blob/bd925525782027969c3812b427aee8d6adddad68/src/tv2-common/hotkeys/hotkey-defaults.ts#L107

The problem before was that multiple items targeted the same sequence of actions with the same indices. I have introduced the ability to transform the target for the pick-filter, based on the position in the list and the type of trigger it is ('queue', 'cut_to_box_XX', etc.).